docs/management_api.yml

```diff
@@ -68,52 +58,6 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
-  /users/inital:
```

Comment:
imo, we don't need to bump API version
do we know about any clients that did integration with this part of API?